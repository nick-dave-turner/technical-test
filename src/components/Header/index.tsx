// @flow
import React from "react";
import styled from "styled-components";

const Container = styled.header`
  position: fixed;
  display: flex;
  flex: 1;
  align-items: center;
  top: 0;
  width: 100%;
  height: 50px;
  padding: 0 5rem;
  background: #ffffff;
  border-bottom: 2px solid #dddddd;
`;

const Title = styled.h1`
  margin: 0;
  padding: 0;
  font-size: 1.5rem;
`;

const Header: React.FC = () => {
  return (
    <Container>
      <Title>Stock Trades</Title>
    </Container>
  );
};

export default Header;
