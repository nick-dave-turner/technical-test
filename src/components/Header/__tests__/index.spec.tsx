import React from "react";
import { shallow } from "enzyme";

import Header from "../";

describe("Header component", () => {
  it("should match snapshot", () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
