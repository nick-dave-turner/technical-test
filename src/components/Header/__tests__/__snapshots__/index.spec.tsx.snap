// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<styled.header>
  <styled.h1>
    Stock Trades
  </styled.h1>
</styled.header>
`;
